packages/manager/apps/hycu/src/tracking.constant.ts

```diff
@@ -1,20 +1,57 @@
+const HYCU_LEVEL2 = '98';
 export const LEVEL2 = {
   EU: {
     config: {
-      level2: '98',
+      level2: HYCU_LEVEL2,
     },
   },
   CA: {
     config: {
-      level2: '98',
+      level2: HYCU_LEVEL2,
     },
   },
   US: {
     config: {
-      level2: '98',
+      level2: HYCU_LEVEL2,
     },
   },
 } as const;
-export const UNIVERSE = 'HostedPrivatedCloud';
-export const SUB_UNIVERSE = 'HostedPrivatedCloud';
-export const APP_NAME = 'hycu';
+export const UNIVERSE = 'PrivatedCloud';
+export const SUB_UNIVERSE = 'storage-backup';
+export const APP_NAME = 'HYCU';
+
+const PREFIX = `${UNIVERSE}::${SUB_UNIVERSE}::${APP_NAME}`;
+export const TRACKING = {
+  breadcrumb: {
+    homeClick: () => `breadcrumb::link::go-to-home`,
+    hycuClick: () => `breadcrumb::link::go-to-hycu`,
+  },
+  onboarding: {
+    beginClick: () => `${PREFIX}::page::button::start_${APP_NAME}`,
```

Comment:
When you don't have any parameter, don't create a function and use a string constant instead.
To apply for all the values
```suggestion
    beginClick: `${PREFIX}::page::button::start_${APP_NAME}`,
```